projects/angular/src/wizard/_wizard.clarity.scss

```
@@ -510,6 +511,22 @@
     }
   }
 
+  .clr-wizard--in-page {
+    // extra specificity is needed here to overcome the following default style without using !important:
+    // .main-container .content-container .content-area > :first-child { margin-top: 0px; }
+    &#{&}#{&}.clr-wizard--in-page-fill-content-area {
```

What's the point ot have `&#{&}#{&}` it will be converted to `.clr-wizard--in-page.clr-wizard--in-page.clr-wizard--in-page-fill-content-area`?

Single `&` is enough.
```suggestion
    &.clr-wizard--in-page-fill-content-area {
```

---

A single `&` is not enough. I explained why in my code comment:

```
// extra specificity is needed here to overcome the following default style without using !important:
// .main-container .content-container .content-area > :first-child { margin-top: 0px; }
```

---

Check the follow up comment with suggestion: https://github.com/vmware-clarity/ng-clarity/pull/1709#discussion_r2063877749